server/operator.go

```diff
@@ -154,28 +154,33 @@ type Operator interface {
 }
 
 type adminOperator struct {
-	Name   string        `json:"name"`
-	Region *RegionInfo   `json:"region"`
-	Start  time.Time     `json:"start"`
-	Ops    []Operator    `json:"ops"`
-	State  OperatorState `json:"state"`
+	Name         string    `json:"name"`
+	Start        time.Time `json:"start"`
+	sync.RWMutex `json:"-"`
```

Comment:
I agree with @siddontang , it's more clear to put the mutex in the beginning to protect the whole structure.